itsy.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>ITSY</title>
+  <link rel="stylesheet" type="text/css" href="styles/itsy.css">
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Amatic+SC:wght@400;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <div class="home">
+    <header>
+      <div class="header__logo amatic">itsy</div>
+        <div class="header-menu amatic">
+          <div class="header__list">home</div>
```

Comment:
seems better `.header-menu__item` or `.header__menu--item`, you khow how it should be.